components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -231,7 +193,7 @@ def main(argv):
         "--aws-secret-access-key", type=str, default=None, help="AWS secret access key."
     )
     s3_compressor_parser.add_argument(
-        "--aws-credentials-file", type=str, default=None, help="Path to AWS credentials file."
+        "--aws-profile", type=str, default=None, help="AWS configuration profile in ~/.aws"
```

Comment:
maybe something like:
```suggestion
        "--aws-profile", type=str, default=None, help="AWS configuration profile to use for authentication"
```

feel free to change it.